### PR TITLE
Update legacy.ts

### PR DESCRIPTION
**Pull request by VoltickVL**
_TZE200_5toc8efa fix settings

### CODE REVIEW

File: src/lib/legacy.ts [hunk 1]
```diff
@@ -6712,15 +6728,16 @@ const toZigbee2 = {
         convertSet: async (entity, key, value: any, meta) => {
             if (['_TZE200_5toc8efa'].includes(meta.device.manufacturerName)) {
                 await sendDataPointValue(entity, dataPoints.moesHeatingSetpoint, value * 10);
-            } else {
-                if (value <= 5) value = Math.round(value*10);
-                await sendDataPointValue(entity, dataPoints.moesHeatingSetpoint, value);
```
Comment (Koenkk) [inline src/lib/legacy.ts:6717]: This changes the behaviour for all except `_TZE200_5toc8efa`, please revert this change.

File: src/lib/legacy.ts [hunk 2]
```diff
@@ -6712,15 +6728,16 @@ const toZigbee2 = {
...
         },
     } as Tz.Converter,
     moes_thermostat_deadzone_temperature: {
         key: ['deadzone_temperature'],
-        convertSet: async (entity, key, value, meta) => {
+        convertSet: async (entity, key, value: any, meta) => {
+            if (['_TZE200_5toc8efa'].includes(meta.device.manufacturerName)) {
+                await sendDataPointValue(entity, dataPoints.moesDeadZoneTemp, value * 10);
+            }
             await sendDataPointValue(entity, dataPoints.moesDeadZoneTemp, value);
```
Comment (Koenkk) [inline src/lib/legacy.ts:6741]: Make this an if else, otherwise you send the value twice, e.g.

```js
            if (['_TZE200_5toc8efa'].includes(meta.device.manufacturerName)) {
                await sendDataPointValue(entity, dataPoints.moesDeadZoneTemp, value * 10);
            } else {
                await sendDataPointValue(entity, dataPoints.moesDeadZoneTemp, value);
            }
```

File: src/lib/legacy.ts [hunk 3]
```diff
@@ -6733,13 +6750,19 @@ const toZigbee2 = {
     } as Tz.Converter,
     moes_thermostat_min_temperature_limit: {
         key: ['min_temperature_limit'],
-        convertSet: async (entity, key, value, meta) => {
+        convertSet: async (entity, key, value: any, meta) => {
+            if (['_TZE200_5toc8efa'].includes(meta.device.manufacturerName)) {
+                await sendDataPointValue(entity, dataPoints.moesMinTempLimit, value * 10);
+            }
             await sendDataPointValue(entity, dataPoints.moesMinTempLimit, value);
```
Comment (Koenkk) [inline src/lib/legacy.ts:6757]: Make if/else as described above

File: src/lib/legacy.ts [hunk 4]
```diff
@@ -6733,13 +6750,19 @@ const toZigbee2 = {
...
         },
     } as Tz.Converter,
     moes_thermostat_max_temperature_limit: {
         key: ['max_temperature_limit'],
-        convertSet: async (entity, key, value, meta) => {
+        convertSet: async (entity, key, value: any, meta) => {
+            if (['_TZE200_5toc8efa'].includes(meta.device.manufacturerName)) {
+                await sendDataPointValue(entity, dataPoints.moesMaxTempLimit, value * 10);
+            }
```
Comment (Koenkk) [inline src/lib/legacy.ts:6765]: Make if/else as described above

Comment (VoltickVL) [reply src/lib/legacy.ts:6770]: thx i edit legacy.ts
